public/js/app/counter.js

```diff
@@ -0,0 +1,26 @@
+const delay = 3000;
+
+$("#game_counter").parent().hide().fadeIn();
+            
+setInterval(function(){
+    $.get('lobby_api', { resource: "players"}, function (body) {
+        let entries = body;
```

Comment:
body could be named as `entries` and this line can be deleted.
I think this variable should be named `players`